twistr/events.go

```diff
@@ -504,18 +506,40 @@ func PlayBrushWar(s *State, player Aff) {
 	   country. On a modified die roll of 3-6, the player receives 1 VP and
 	   replaces all the opponent’s Influence in the target country with their
 	   Influence. The player adds 3 to its Military Operations Track. */
+	stabLTE := func(c *Country) error {
+		if c.Stability > 2 {
+			return fmt.Errorf("Country needs to have stability of 1 or 2")
+		}
+		return nil
+	}
+
+	cl := SelectInfluenceForce(s, player, func() ([]*Country, error) {
```

Comment:
Resolved. We're not listing all those countries in SelectCountry. Keeping as is for now.